internal/sdkprovider/service/vpc/project_vpc.go

```diff
@@ -22,13 +22,13 @@ var aivenProjectVPCSchema = map[string]*schema.Schema{
 		ForceNew:    true,
 		Required:    true,
 		Type:        schema.TypeString,
-		Description: userconfig.Desc("Defines where the cloud provider and region where the service is hosted in. See the Service resource for additional information.").ForceNew().Build(),
+		Description: userconfig.Desc("The cloud provider and region where the service is hosted in the format `CLOUD_PROVIDER-REGION_NAME`. For example, `google-europe-west1` or `aws-us-east-2`. You can get this value from the service resource.").ForceNew().Build(),
```

Comment:
I think, "You can get this value from the service" is misleading. Usually, a service runs on a virtual private network, not vice-versa. So probably, the old sentence pointed to the resource description for an [additional information](https://registry.terraform.io/providers/aiven/aiven/latest/docs/resources/kafka#cloud_name). Probably, we used to have "a service" resource one for all kafka, pg, etc, or it's been just copy-pasted from the API doc.

Reply:
Could be as there is a service endpoint. I've removed the sentence since I don't think it's helpful and we don't use this type of thing in other descriptions.